### DISCUSSION TITLE
Importing QFX

### DISCUSSION THREAD

**Discussion by rphillipchuk** [actualbudget/releases#138] [Q&A] [answered]
QFX File

I am confused........I am starting fresh, so I go to my Bank and download the QFX for the checking account..... I create a checking Account in Actual with the balance of

$2272.70 which is my current balance.

After the QFX import, it shows a balance of

$2045.57

Please advise

**Comment by bpaulien** [comment, marked as answer]
So, your starting balance was $2272.70?  Is that what you put in as starting balance when you created the account in Actual?  Did you import $227.13 worth of transactions?  I'm guessing you need to adjust your starting balance.  You probably imported in some transactions that were actually included in that starting balance.  

So I guess you could do 1 of 2 things.

1.) Adjust your starting balance (adjust it by $227.13) so the new transactions bring the "current" or cleared balance up to $2272.70.
OR
2.) delete the transactions, and start with the starting balance of $2272.70, and only import transactions that happen AFTER the date of your beginning balance... If you do this, you might want to adjust the budgeted categories to show those as already having been paid.

ETA:  If you're starting today, but want to include transactions from the 1st, then you need to have your starting balance be what it was on the 1st, so those transactions between then and now will be accounted for in the current balance.

Edited 1 more time...  You'll actually want to set the balance to what it was at the end of Nov 30th.  So transactions from the 1st will adjust  your running balance accordingly.

**Reply by rphillipchuk** [reply]
Great advice
